code/io/joy-sdl.cpp

```diff
@@ -226,6 +226,10 @@ void setPlayerJoystick(Joystick* stick, short cid)
 	Assert((cid >= CID_JOY0) && (cid < CID_JOY_MAX));
 	pJoystick[cid] = stick;
 
+	if (cid == CID_JOY0) {
```

Comment:
Is force feedback intended to only be supported for `JOY0`?

Reply:
Yes, it's always been limited to `JOY0`. The code really wasn't set up by V to work with more than one haptic device to begin with.